entproto/internal/todo/ent/schema/user.go

```diff
@@ -87,24 +87,29 @@ func (User) Fields() []ent.Field {
 		field.Bool("opt_bool").
 			Optional().
 			Annotations(entproto.Field(15)),
+		field.Strings("opt_strings").
```

Comment:
with protos, you always add new fields with new tag numbers, never replace existing ones. lets do that in the tests as well